test/Microsoft.AspNetCore.SystemWebAdapters.Tests/HttpRequestTests.cs

```diff
@@ -811,5 +811,30 @@ public void ReadMoreBytes()
             // Arrange
             Assert.True(bytes.SequenceEqual(bytesRead));
         }
+
+        [Fact]
+        public void AcceptTypes()
+        {
+            // Arrange
+            var headers = new HeaderDictionary
+            {
+                { HeaderNames.Accept, "text/html, application/xml;q=0.9, */*;q=0.8, application/xhtml+xml" }
+            };
+
+            var requestCore = new Mock<HttpRequestCore>();
+            requestCore.Setup(r => r.Headers).Returns(headers);
+
+            var request = new HttpRequest(requestCore.Object);
+
+            // Act
+            var acceptTypes = request.AcceptTypes;
+
+            // Assert
+            Assert.Collection(acceptTypes,
+                a => Assert.Equal("text/html", a),
+                a => Assert.Equal("application/xml", a),
+                a => Assert.Equal("*/*", a),
+                a => Assert.Equal("application/xhtml+xml", a));
+        }
```

Comment:
Nit-pick - it might be good to add a test for empty or missing accept headers since that scenario could regress without breaking this case.